src/lib/process/index.ts

```diff
@@ -243,7 +243,7 @@ export const disableProcess = async (
 ): Promise<Process.ProcessResponse> => {
   const processId = utf8ToHex(name)
 
-  const polkadot: Polkadot.Polkadot = await createNodeApi(options)
+  await using polkadot = await createNodeApi(options)
```

Comment:
so the `using` uses this (from he file below): 
```
[Symbol.asyncDispose]: async () => {
      await api.disconnect()
    },
```
to dispose of the polkadot instance/disconnect it once this block finishes? (that's the explanation I found)

Reply:
the syntax is terrifying but the behaviour is awesome